internal/provider/resource_service.graphql

```diff
@@ -51,8 +53,11 @@ query getServiceInstance(
   }
 }
 
-# @genqlient(for: "ServiceInstanceUpdateInput.cronSchedule", pointer: true)
-# @genqlient(for: "ServiceInstanceUpdateInput.source", pointer: true)
+# @genqlient(for: "ServiceInstanceUpdateInput.cronSchedule", omitempty: true, pointer: true)
```

Comment:
The cronschedule issue might be to not set `omitempty` here.

Reply:
Yeah, I got the idea